lib/handlers/quote/quote.ts

```diff
@@ -459,17 +462,60 @@ export class QuoteHandler extends APIGLambdaHandler<
       simulationError: simulationStatus == SimulationStatus.Failed,
       gasPriceWei: gasPriceWei.toString(),
       route: routeResponse,
-      routeString: routeAmountsToString(route),
+      routeString,
       quoteId,
     }
 
     metric.putMetric(`GET_QUOTE_200_CHAINID: ${chainId}`, 1, MetricLoggerUnit.Count)
+
+    this.logRouteMetrics(metric, currencyIn, currencyOut, type, chainId, amount, routeString)
+
     return {
       statusCode: 200,
       body: result,
     }
   }
 
+  private logRouteMetrics(
+    metric: IMetric,
+    currencyIn: Currency,
+    currencyOut: Currency,
+    tradeType: 'exactIn' | 'exactOut',
+    chainId: number,
+    amount: CurrencyAmount<Currency>,
+    routeString: string
+  ): void {
+    const PAIRS_TO_TRACK = [
+      'WETH/USDC',
+      'USDC/WETH',
+      'USDT/WETH',
+      'WETH/USDT',
+      'USDC/USDT',
+      'USDT/USDC',
+      'WMATIC/USDC',
+      'USDC/WMATIC',
+    ]
```

Comment:
Its best to always use addresses since tokens can use the same symbol

Reply:
The addresses can be different on different chains though, so need to handle that

Reply:
I think we are okay with that collision for now, these metrics are mostly to gather some initial data but I wouldn't keep them around forever.

Would it be okay if we kept this way for now, with the understanding that tokens could use the same symbol? I think the likelihood of that happening is low given the pairs being used.